tensorboard/webapp/runs/views/runs_table/runs_data_table.ng.html

```diff
@@ -15,6 +15,44 @@
 limitations under the License.
 -->
 
+<custom-modal #contextMenu>
```

Comment:
I think this should be in the data table.

Reply:
Done